hydrogram/methods/phone/discard_grou_call.py

```diff
@@ -0,0 +1,78 @@
+#  Hydrogram - Telegram MTProto API Client Library for Python
```

Comment:
Here the file has a typo in the name, it says discard_grou_call, instead of discard_group_call